package.json

```diff
@@ -7,17 +7,22 @@
     "doc": "docs",
     "test": "test"
   },
-  "dependencies": {},
+  "dependencies": {
+    "react": "0.13.3"
+  },
   "devDependencies": {
     "babel-cli": "^6.8.0",
+    "babel-loader": "6.2.8",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
     "eslint": "^3.1.1",
     "eslint-config-google": "^0.6.0",
     "eslint-plugin-react": "^5.2.2",
     "grunt": "^1.0.1",
     "grunt-babel": "^6.0.0",
-    "load-grunt-tasks": "^3.5.0"
+    "grunt-run": "0.6.0",
```

Comment:
It seems that this add-on can run any 'bash' command. Does it mean we can get rid of `grunt-babel` and just use run for this?